tests/lib/transforms.js

```diff
@@ -82,41 +83,73 @@ function testTransformWithFixtures(transform, fixtures) {
     });
 }
 
-describe("New Rule Format transform", function() {
-    testTransformWithFixtures(newRuleFormatTransform, [
-        // tests basic functionality of the transform
-        "simple",
-
-        // tests that the transform can detect that a rule is fixable
-        "fixable",
+/**
+ * Determines if the file name is *.input.js
+ *
+ * @param {string} name - file name
+ * @returns {boolean} - does it match?
+ * @private
+*/
+function inputsFilter(name) {
+    return /\.input\.js/.test(name);
```

Comment:
I think this regex should end with an end-of-string anchor (`$`).

Reply:
Good catch, thanks!